src/Core/Addresses/Models/Address.php

```diff
@@ -14,43 +15,43 @@ class Address extends BaseModel
      */
     protected $hashids = 'user';
 
+    protected $dates = [
+        'last_used_at'
+    ];
+
     /**
      * The attributes that are mass assignable.
      *
      * @var array
      */
-    protected $fillable = [
-        'address',
-        'address_three',
-        'address_two',
-        'billing',
-        'city',
-        'country',
-        'county',
-        'default',
-        'firstname',
-        'lastname',
-        'shipping',
-        'state',
-        'zip',
-    ];
+    protected $guarded = [];
 
     public function getFieldsAttribute()
```

Comment:
What is this function used for? My reaction is it shouldn't be a "get" method.

Reply:
It's an accessor for the fields attribute. The naming is an Eloquent convention

Reply:
It's supposed to be used for actual database fields though, not misc. functions.

Reply:
I see what you're looking at now tbh I haven't focused on that on this run so would need to refresh myself on it.

Reply:
Leave this, it's fine.